web/tests/settings_org.test.cjs

```diff
@@ -61,14 +65,14 @@ function createSaveButtons(subsection) {
     const $save_button_controls = $(".save-button-controls");
     const $stub_save_button = $(".save-discard-widget-button.save-button");
```

Comment:
We do not use `save-discard-widget-button` anymore, so maybe we can update them here as well.

Reply:
Updated the selectors to just `.save-button` and `.discard-button`.